### PR TITLE
Fix class selection in navigation pane

### PR DESCRIPTION
**Pull request by astigsen**
Fixes issues #114 and #115. I have no experience with OS X programming so please review carefully ;-)

### CODE REVIEW

File: RealmBrowser/Classes/RLMViewController.m [hunk 1]
```diff
@@ -71,9 +71,14 @@ - (void)setSelectionIndex:(NSUInteger)newIndex
 {
     NSUInteger oldIndex = self.tableView.selectedRow;
     if (oldIndex != newIndex) {
-        [self.tableView selectRowIndexes:[NSIndexSet indexSetWithIndex:newIndex] byExtendingSelection:NO];
-        
-        [self.tableView scrollRowToVisible:newIndex];
+        if (newIndex == (NSUInteger)-1) {
```
Comment (TimOliver) [inline RealmBrowser/Classes/RLMViewController.m]: It might be more appropriate to use `NSUIntegerMax` instead of typecasting -1 there. :)

**Comment by TimOliver** [comment]
Thanks a lot Alexander!

I tested it out here and it all worked great for me. I left my one piece of feedback in the comments.

I'm happy to merge if there are no other objections.

**Comment by astigsen** [comment]
Cool. I fixed that. Merge at will :-)

File: RealmBrowser/Classes/RLMRealmBrowserWindowController.m [hunk 2]
```diff
@@ -101,6 +95,12 @@ -(void)realmDidLoad
     
     [self reloadAfterEdit];
     self.window.alphaValue = 1.0;
+
+    id firstItem = self.modelDocument.presentedRealm.topLevelClasses.firstObject;
+    if (firstItem != nil) {
+        RLMNavigationState *initState = [[RLMNavigationState alloc] initWithSelectedType:firstItem index:-1];
```
Comment (jpsim) [inline RealmBrowser/Classes/RLMRealmBrowserWindowController.m]: `-1` should be `NSNotFound`

File: RealmBrowser/Classes/RLMViewController.m [hunk 3]
```diff
@@ -71,9 +71,14 @@ - (void)setSelectionIndex:(NSUInteger)newIndex
 {
     NSUInteger oldIndex = self.tableView.selectedRow;
     if (oldIndex != newIndex) {
-        [self.tableView selectRowIndexes:[NSIndexSet indexSetWithIndex:newIndex] byExtendingSelection:NO];
-        
-        [self.tableView scrollRowToVisible:newIndex];
+        if (newIndex == NSUIntegerMax) {
```
Comment (jpsim) [inline RealmBrowser/Classes/RLMViewController.m]: `NSUIntegerMax` should be `NSNotFound`

**Pull request merged**